src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -818,8 +818,26 @@ public void pulse() {
             }
         }
 
-        if (foodLevel == 0 && getHealth() > 1 && ticksLived % 80 == 0) {
-            damage(1, DamageCause.STARVATION);
+        // Process food level and starvation based on difficulty.
+        if (world.getDifficulty().equals(Difficulty.PEACEFUL)) {
```

Comment:
Use a switch statement.

Reply:
In the ```default``` part of the ```switch```, should I leave it blank or put something?

Reply:
Yes, just add a comment that says ` // do nothing` or something similar.